generator/config/search/autocomplete.yaml

```
@@ -0,0 +1,152 @@
+# $schema: ../schema.json
+name: autocomplete
+link: 'https://www.mongodb.com/docs/atlas/atlas-search/autocomplete/'
+type:
+    - searchOperator
+encode: object
+description: |
+    The autocomplete operator performs a search for a word or phrase that
+    contains a sequence of characters from an incomplete input string. The
+    fields that you intend to query with the autocomplete operator must be
+    indexed with the autocomplete data type in the collection's index definition.
+arguments:
+    -
+        name: path
+        type:
+            - searchPath
+    -
+        name: query
+        type:
+            - string
+    -
+        name: tokenOrder
+        optional: true
+        type:
+            - string #tokenOrder
```

Does this comment add any value? I assume it may be referring to:

```
"tokenOrder": "any|sequential",
```

I assume you don't want to define an enum for that, nor enforce it client-side, but are the possible values documented anywhere in the builder?

---

Updating the comment, as I don't have implemented this enum for now.
```suggestion
            - string # any|sequential
```